src/components/button/button.svelte

```diff
@@ -43,14 +45,19 @@
 
   export let kind: Props.ButtonKind = 'filled'
   export let size: Props.ButtonSize = 'medium'
-  export let isLoading: boolean = false
+  export let isLoading: boolean | string = false
   export let isDisabled: Disabled = undefined
   export let href: Href = undefined
   export let fab = false
 
   export let onClick: () => void = undefined
 
   $: tag = href ? 'a' : ('button' as 'a' | 'button')
+
+  // Web components set these attributes to the empty string, which is falsey
+  // so we need to consider the empty string as truthy.
+  $: loading = !!(isLoading || isLoading === '')
+  $: disabled = !!(isDisabled || isDisabled === '')
```

Comment:
Is this unfortunate that we need to make this consideration for every boolean prop on every component? Can we abstract it away?

Reply:
There probably is - unfortunately it'd require us pulling out the type of the prop at build time and treating them differently in the webcomponents wrapper. 

Doable, but probably not super quick.

I had a bit more of a think about why we haven't encountered this before:

At runtime, we check the value of the props and if they're a boolean, we treat that attribute as a boolean from then on. Unfortunately, `isDisabled` is `undefined` by default.

I've removed the wrapping for isLoading, based on this.